src/EA.Weee.Web.Tests.Unit/Areas/Admin/Controllers/ProducerSubmissionControllerTests.cs

```diff
@@ -320,10 +324,10 @@ public async Task RemoveSubmission_Get_ReturnAndPopulatesViewModel()
             result.Model.Should().NotBeNull();
 
             var viewModel = result.Model as ConfirmRemovalViewModel;
-
+            
```

Comment:
@stevelee-civica there should be two tests for this. One when the authed rep has data and one when not.

Reply:
added